telethon_examples/payment.py

```diff
@@ -0,0 +1,143 @@
+from telethon import TelegramClient, events, types, functions
+
+import asyncio
+import logging
+import tracemalloc
+
+loop = asyncio.get_event_loop()
+
+api_id = 0
+api_hash = ""

```

Comment:
Please do the same as https://raw.githubusercontent.com/LonamiWebs/Telethon/master/telethon_examples/replier.py (use `get_env`), at least to configure these.